Update Lisk Service setup, config, management, index and references - Closes #678 (#679)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/docker.adoc

:url_docker_install_linux_compose: https://docs.docker.com/compose/install/
:url_docker_install_mac: https://docs.docker.com/docker-for-mac/install/
:url_docker_install_windows: https://docs.docker.com/docker-for-windows/install/
:url_docker_linux_post_install: https://docs.docker.com/install/linux/linux-postinstall/
:url_xcode: https://developer.apple.com/xcode/features/

:url_index_usage: index.adoc#usage
:url_setup: setup/index.adoc
:url_config: configuration/docker.adoc
:url_management: management/docker.adoc

This guide describes how to setup Lisk Service with Docker.
Alternative options of how to install Lisk Service are described on the xref:{url_setup}[Setup] page.

== Prerequisites

The following dependencies are required to install and run Lisk Service with Docker.

=== Development tools

[tabs]
====
Ubuntu / Debian::
+
--
Install the `build-essential` package alongside with several development tools.

For example, use `apt` if running Ubuntu as shown in the following command below:

[source,bash]
----
apt install build-essential git
----
--
MacOS::
+
--

*Brew*

Install https://brew.sh/[Brew] by following https://brew.sh/[the most recent instruction].

> If you already have Brew installed, ensure it is working and up-to-date.

Reviewer: ```suggestion
If you already have Brew installed, ensure it is working and is the latest version.
```
